.travis.yml

```diff
@@ -4,5 +4,10 @@ jdk:
 before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
+
+script:
+  - mvn javadoc:jar
+  - mvn javadoc:test-aggregate
```

Comment:
I'm not sure we can configure it... it's a step it perform for all java maven projects before it runs any of the configured scripts.  If you don't configure any scripts it runs "mvn test -B", otherwise it runs whatever your scripts are.  Since we liked that it ran "mvn test -B", I think adding that to our list is good.  It does seem silly that we install skipping tests and javadoc then we build again for javadocs and tests, but perhaps it compartmentalizes the types of errors it might report.